From f73a8fadab3c4c369ee9e26f85d9d79e25a32ccc Mon Sep 17 00:00:00 2001
From: Piyush Gupta <gpiyush@vmware.com>
Date: Wed, 19 Apr 2023 13:18:31 +0000
Subject: [PATCH 02/14] sample_ks_ostree_client_from_{default,server}.cfg:
 Allow RootLogin in postinstall.

Set PermitRootLogin to "yes" in postinstall.

Change-Id: Ieb649753126aad36b37041f32abd2422a7bbfdb3
---
 sample_ks/sample_ks_ostree_client_from_default.cfg | 4 ++++
 sample_ks/sample_ks_ostree_client_from_server.cfg  | 7 ++++++-
 2 files changed, 10 insertions(+), 1 deletion(-)

diff --git a/sample_ks/sample_ks_ostree_client_from_default.cfg b/sample_ks/sample_ks_ostree_client_from_default.cfg
index 23ad3d2..a3e8958 100644
--- a/sample_ks/sample_ks_ostree_client_from_default.cfg
+++ b/sample_ks/sample_ks_ostree_client_from_default.cfg
@@ -8,5 +8,9 @@
   "ostree": {
     "default_repo": true
   },
+  "postinstall":[
+    "#!/bin/sh",
+    "sed -i 's/.*PermitRootLogin.*/PermitRootLogin yes/' /etc/ssh/sshd_config"
+  ],
   "photon_docker_image": "photon:5.0"
 }
diff --git a/sample_ks/sample_ks_ostree_client_from_server.cfg b/sample_ks/sample_ks_ostree_client_from_server.cfg
index 455adfb..6585c36 100644
--- a/sample_ks/sample_ks_ostree_client_from_server.cfg
+++ b/sample_ks/sample_ks_ostree_client_from_server.cfg
@@ -8,8 +8,13 @@
     "disk": "/dev/sda",
     "ostree": {
                 "default_repo": false,
-                "repo_url": "http://127.0.0.0:8000/repo",
+                "repo_url": "https://packages.vmware.com/photon/rpm-ostree/base/5.0/x86_64/repo/",
                 "repo_ref": "photon/5.0/x86_64/minimal"
               },
+    "postinstall":
+        [
+            "#!/bin/sh",
+            "sed -i 's/.*PermitRootLogin.*/PermitRootLogin yes/' /etc/ssh/sshd_config"
+        ],
     "photon_docker_image": "photon:5.0"
 }
-- 
2.23.1

